apps/ai-image-generator/package.json

```diff
@@ -19,8 +19,11 @@
   "dependencies": {
     "@contentful/app-scripts": "1.13.1",
     "esbuild": "^0.23.0",
-    "sharp": "^0.33.0",
     "node-addon-api": "^7.0.0",
-    "node-gyp": "^10.0.1"
+    "node-gyp": "^10.0.1",
```

Comment:
these were env fixes required for the general build to work. I'm guessing react-scripts included `lru-cache` in its tree somehow